### ISSUE TITLE
Missing each/range implementations

### ISSUE THREAD

**Issue by LauJensen** [i-net-software/jlessc#47]
Are these scheduled for inclusion?

**Comment by Horcrux7** [comment]
Currently there is no plan for it. Do you need it for a specific framework to working? If yes, which?

**Comment by LauJensen** [comment]
No, we've just been handed some LESS files from a designer and they break at the each/range lines. We've built a small pre-processor to side-step the issue, but would prefer to see it in master.
